docker-compose.yml

```diff
@@ -38,12 +38,13 @@ services:
       - .env
       - .env-advanced
     environment:
-      - "NATS_URL=nats://nats:4222"
-      - "LAMBDAS_URL=http://lambdas:7070"
-      - "CONTENT_URL=http://content-server:6969"
-      - "LIGHTHOUSE_URL=http://comms-server:9000"
+      - NATS_URL=nats://nats:4222
+      - LAMBDAS_URL=${CATALYST_URL}/lambdas/
+      - CONTENT_URL=${CATALYST_URL}/content/
+      - LIGHTHOUSE_URL=${CATALYST_URL}/comms/
```

Comment:
I think it is better to use the internal NS, to prevent going to cloudflare, and through the nginx to talk to lambdas

Reply:
yes, but this is used for the public urls, @agusaldasoro is building a PR to split those